XCommas.Net/XCommas.Net/Objects/User.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace XCommas.Net.Objects
+{
+    public enum User
```

Comment:
This enum should be named UserMode to better convey what it does. Enum members should be properly cased (ie. they must start with uppercase).